src/document.c

```diff
@@ -757,10 +761,21 @@ static PreprocessorFunc preprocessorMap[] = {
     [IXFLDPOS_GEOMETRY] = geometryPreprocessor,
     };
 
+
 int IndexerBulkAdd(RSAddDocumentCtx *cur, RedisSearchCtx *sctx,
                    const DocumentField *field, const FieldSpec *fs, FieldIndexerData *fdata,
-                   QueryError *status) {
+                   QueryError *status) {  
+  
+  static size_t opCounter = 0;
```

Comment:
Can't we expose `opCounter` for the debug command instead of having both `opCounter` and `g_yieldCallCounter`?